### ISSUE TITLE
No coverage changes even though base branch & head branch has different code coverage

### ISSUE THREAD

**Issue by LeoYuan** [ArtiomTr/jest-coverage-report-action#285]
### Describe a bug

I comment some unit-test code on head branch, then make a pull request to base branch. The coverage action has been triggered as expected, and coverage comment has been made.

But, there is no coverage changes in action comment, see below plz.
![image](https://user-images.githubusercontent.com/1195765/174134430-da98a911-5c7f-4aac-a4c0-107c1ad2ab6b.png)

### Expected behavior

Shows coverage changes correctly~

### Details

-   Action version: v2.0
-   OS, where your action is running (windows, linux):
-   <details><summary>action.yml file</summary>

        ```yml
        cov-designer:
          runs-on: ubuntu-latest
          # skip fork's PR, otherwise it fails while making a comment
          if: ${{ github.event.pull_request.head.repo.full_name == 'alibaba/lowcode-engine' }}
          steps:
            - name: checkout
              uses: actions/checkout@v2
      
            - uses: actions/setup-node@v2
              with:
                node-version: '14'
      
            - name: install
              run: npm i && npm run setup:skip-build
      
            - uses: ArtiomTr/jest-coverage-report-action@v2
              with:
                working-directory: packages/designer
                test-script: npm test -- --jest-ci --jest-json --jest-coverage --jest-testLocationInResults --jest-outputFile=report.json
                package-manager: yarn
                annotations: none
        ```

    </details>

-   <details><summary>Screenshots</summary>

        

        

    </details>

### Additional context

Here is my test [PR](https://github.com/alibaba/lowcode-engine/pull/665).

**Comment by LeoYuan** [comment]
Oh man, finally I found why there is no coverage percentage delta...

You set the minimum delta threshold with 1%. 😭
https://github.com/ArtiomTr/jest-coverage-report-action/blob/3e4f3e2f5f42b2eb5d7c48280ed8327979f4bb4a/src/utils/formatPercentage.ts#L7-L26

**Comment by ArtiomTr** [comment]
Hey @LeoYuan :wave:,

I've added this to remove unnecessary noise from the report. I thought it was a good idea, as sometimes coverage information changes so slightly, that it doesn't require additional attention. However, now I see that this can be misleading behaviour. I will remove this approximation threshold.

**Comment by LeoYuan** [comment]
Agreed, it's a better idea to let code owner knows what's happened so they can make a decision to accept / reject PR.
In this case, 1% is a 'big' number.😃

**Comment by regan-karlewicz** [comment]
I agree with @LeoYuan. If you have a large amount of files, 1% may be a large amount of test coverage. The user of this action should be able to specify the threshold.

**Comment by ayansome1** [comment]
Would really apprecaite if this could be prioritized.

**Comment by ArtiomTr** [comment]
Fixed this, fix is available under the `v2.1.1` tag or just `v2`

**Comment by ayansome1** [comment]
Thanks a lot. Great work.
